diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/PositionInTextLocator.kt

```diff
@@ -0,0 +1,70 @@
+package org.cqfn.diktat.ruleset.utils
```

Comment:
Please add some sort of fixme comment here that this code is copy-pasted from ktlint and we might change it in the future.